test/e2e/storage/csi_volumes.go

```diff
@@ -28,7 +28,6 @@ import (
 var csiTestDrivers = []func() testsuites.TestDriver{
 	drivers.InitHostPathCSIDriver,
 	drivers.InitGcePDCSIDriver,
-	drivers.InitHostPathV0CSIDriver,
```

Comment:
We may need to remove this from 1.16 test suites too, otherwise 1.16->1.17 upgrade jobs are going to fail

Reply:
https://github.com/kubernetes/kubernetes/pull/84811